banzai/dbs.py

```diff
@@ -294,99 +213,23 @@ def add_or_update_record(db_session, table_model, equivalence_criteria, record_a
     return record
 
 
-def populate_calibration_table_with_bpms(directory, runtime_context):
```

Comment:
Do you have another way to add BPMs to the calibration table now? The reason I ask is because when commissioning a new instrument, we use this functionality to add in a BPM that's dropped into `/cache` by someone like Nikolaus.

See: https://sites.google.com/lco.global/docs/data-pipelines/banzai-data-pipeline?authuser=0#h.p_5pyUK_Txlq8V

Reply:
Yes. There is a command line script that does this.